opencti-platform/opencti-front/lang/back/de.json

```diff
@@ -19,6 +19,8 @@
   "Actions": "Aktionen",
   "Active": "Aktiv",
   "Activity listeners IDs": "IDs von Aktivitätshörern",
+  "Add in generated & existing indicators in the container": "Hinzufügen von generierten & vorhandenen Indikatoren im Container",
+  "Add in generated & existing observables in the container": "Hinzufügen von generierten & vorhandenen Observablen im Container",
```

Comment:
"add in ... in container" ? is it the right message ? not "add ... in the container" ? to check with product